tsconfig-base.json

```diff
@@ -5,6 +5,7 @@
     "jsx": "react",
     "module": "ES2020",
     "moduleResolution": "Node",
+    "resolveJsonModule": true,
```

Comment:
Nice. Any reason not to turn this on and do the same in `@workos-inc/node` for the [`VERSION` constant](https://github.com/workos/workos-node/blob/80edc61076eb9a2749756051a083a947198497cf/src/workos.ts#L34)?